redaxo/src/addons/phpmailer/update.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+if ($this->getConfig('backup') == '') {
+    $this->setConfig('backup', 1);
+}
```

Comment:
Bei jedem Update werden nun also die Backups wieder aktiviert? ;)
Besser mit Versions-Check.

Reply:
nein.. nur wenn nicht vorhanden :)

Reply:
auch wenn 'backup' gleich `0`, denn `0 == ''` ergibt true: https://3v4l.org/XfAXs

Reply:
Passt immer noch nicht, jetzt hast du den typsicheren Vergleich, `''` kommt aber nie raus, somit greift das if nie.
Wenn leer, kommt `null` raus. Besser ist allerdings einfach `$this->hasConfig('backup')` zu nutzen. ;)